### ISSUE TITLE
New servers, HX and FE? Also doesn't always get links and returns htm file?

### ISSUE THREAD

**Issue by bearachute** [thorio/KGrabber#43]
I love the script, this is the third iteration of a script for the kim/kiss cartoon family but it seems since openload is gone, and other servers aren't working right they have a few new ones out. There's alpha and 2 betas and FE and HX. When I try to grab links from these sites it just returns a htm file which obviously isn't the right thing. The most success I have from this is using IDM to download file individually, or click on the links it produces in seperate tabs. 

So I guess what I'm saying is I love the script it just isn't working great?

Chrome
Windows 10
Tampermonkey
Any show really on Kimcartoon, tried Wabbit, Gumball, and Rise of the TMNT. 

I noticed you used some node for this script, is it possible to figure out how to get these links that way? Or maybe a different way? There are so many of these scripts around in different languages I'd prefer your way (easier, don't have to drop Selenium in there or anything) but if there's any other way to grab these links and download I'd be down for it.

Thanks

**Comment by thorio** [comment]
I rarely use kimcartoon, so it's no wonder those parts of the script are outdated.
Beta seems to work but someone else said it's only bad quality.
Some poking at Beta 2 reveals they're using HLS, which I've failed at before (see #35). The best I could do there is decrypt the playlist files. VLC won't play them, and I'm not sure what is actually needed to make them work.

While looking at the others however, i noticed that FE seems suspiciously similar to Nova, and indeed they expose the same API and even share the same videos.

It's funny, really. I was just poking at the site and now I found a solution that literally just requires me to enable existing functionality for a different server.
The changes will be coming your way soon. 6360be259d5f0358305e877dbcea4d923d9f2479
